rero_ils/modules/items/api/__init__.py

```diff
@@ -0,0 +1,158 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""API for manipulating items."""
+
+
+from functools import partial
+
+from elasticsearch.exceptions import NotFoundError
+from invenio_search import current_search
+
+from .circulation import ItemCirculation
+from .record import ItemRecord
+from ..models import ItemIdentifier
+from ...api import IlsRecordError, IlsRecordsIndexer, IlsRecordsSearch
+from ...documents.api import Document, DocumentsSearch
+from ...fetchers import id_fetcher
+from ...minters import id_minter
+from ...providers import Provider
+
+# provider
+ItemProvider = type(
+    'ItemProvider',
+    (Provider,),
+    dict(identifier=ItemIdentifier, pid_type='item')
+)
+# minter
+item_id_minter = partial(id_minter, provider=ItemProvider)
+# fetcher
+item_id_fetcher = partial(id_fetcher, provider=ItemProvider)
+
+
+class ItemsSearch(IlsRecordsSearch):
```

Comment:
I think that the primary use of __init__.py is to initialize Python packages. Is it possible to move class somewhere else? maybe into `helpers.py` or why not into `api/record.py`?

Reply:
`__init__.py` can contains code. This is not a problem. But for more clarity we can put in `api/api.py`, but this require to defined an `__all__` variable. Putting this in the helpers is not a good idea, nor in record as the Item inherit from two classes.